udb3-ui/udb3-button.stories.js

```diff
@@ -0,0 +1,30 @@
+import Udb3Button, { ButtonVariant } from './udb3-button.vue';
```

Comment:
Just as an FYI, Storybook also supports the `MDX` format for stories as an alternative to JS. Might be a bit easier to write stories, but I have no strong preference if you both prefer pure JS. Just throwing it out there so we can decide now on a format before we have a lot of stories to convert to one or the other. https://storybook.js.org/docs/vue/api/mdx

Reply:
Javascript is life though ❤️ But in all seriousness, also could be an option 🙂

Reply:
Haha 😃 It seems to require a bit less boilerplate code at first glance, but maybe the best way to decide would be to rewrite these same stories in MDX and compare them.